apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainInOrderOnlyGroupedCreatorSamples.kt

```diff
@@ -2,8 +2,239 @@ package ch.tutteli.atrium.api.fluent.en_GB.samples
 
 import ch.tutteli.atrium.api.fluent.en_GB.*
 import ch.tutteli.atrium.api.verbs.expect
+import ch.tutteli.atrium.logic.utils.Group
+import ch.tutteli.atrium.creating.Expect
 import kotlin.test.Test
 
 class IterableLikeToContainInOrderOnlyGroupedCreatorSamples {
 
+    // Helper functions to create Group instances for values
+    private fun <T> groupOf(vararg elements: T): Group<T> = object : Group<T> {
+        override fun toList(): List<T> = elements.toList()
+    }
+
+    // Helper functions to create Group instances for entries (expectation lambdas)
+    private fun <T> groupOfEntries(vararg elements: (Expect<T>.() -> Unit)?): Group<(Expect<T>.() -> Unit)?> = object : Group<(Expect<T>.() -> Unit)?> {
+        override fun toList(): List<(Expect<T>.() -> Unit)?> = elements.toList()
+    }
```

Comment:
thanks for the idea but samples shouldn't introduce auxiliary constructs, they should use only what 
already exists in Atrium. Atrium has Value/Values and Entry/Entries data classes which are `Group`s. 
See 
apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableToContainInOrderOnlyGroupedValuesExpectationsSpec.kt => ambiguity test

and apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableToContainInOrderOnlyGroupedEntriesExpectationsSpec.kt => ambiguity test

I'll make an example in another comment